File: 2st/Riyeon/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        int answer = 0;
        String[] s = binomial.split(" ");
        if(s[1].equals("+"))
            answer = Integer.parseInt(s[0]) + Integer.parseInt(s[2]);
        else if(s[1].equals("-"))
            answer = Integer.parseInt(s[0]) - Integer.parseInt(s[2]);
        else
            answer = Integer.parseInt(s[0]) * Integer.parseInt(s[2]);
        return answer;
    }
}

File: 4st/Seohyun/programmer/간단한 식 계산하기.js
function solution(binomial) {
  let num = binomial.split(" ");

  let n1 = parseInt(num[0]);
  let n2 = parseInt(num[2]);
  let op = num[1];

  if (op === "+") return n1 + n2;
  else if (op === "-") return n1 - n2;
  else if (op === "*") return n1 * n2;
}
